### ISSUE TITLE
[Enhancement] Support different output formats (e.g. JSON)

### ISSUE THREAD

**Issue by akittana** [aws-cloudformation/cloudformation-guard#49]
**Is your feature request related to a problem? Please describe.**
Currently, it is hard to parse the output of cloudformation-guard using other scripts since the output is in text only, having different output formats would help ingest that output to different tools.

**Describe the solution you'd like**
It will be nice to have an option like [`cfn-lint`](https://github.com/aws-cloudformation/cfn-python-lint/#parameters)/[`cfn_nag`](https://github.com/stelligent/cfn_nag#usage)/[`cfripper`](https://github.com/Skyscanner/cfripper#using-json-format-and-output-folder-argument)/[`tflint`](https://github.com/terraform-linters/tflint#usage)/[`tfsec`](https://github.com/tfsec/tfsec#output-options)/[`terrascan`](https://github.com/accurics/terrascan#terrascan-commands)/[`checkov`](https://github.com/bridgecrewio/checkov#features) that allows the user to specify different output options

**Describe alternatives you've considered**
Current solution is to parse the text output which is not ideal

**Comment by nathanataws** [comment]
This shouldn't be too hard to pull together.  I can just use serde logic that's in the lambda to give you the same structured result from the CLI.

**Comment by PatMyron** [comment]
This would be useful for IDE/editor/code review bot integrations

**Comment by sfarazkazmi** [comment]
+1 on this. Text parsing is not ideal and we need JSON based output.

**Comment by jotompki** [comment]
As mentioned in #111, rule evaluation output in the form of JSON is targeted for the next release

**Comment by tvb** [comment]
@jotompki does this mean it will clear the road for https://github.com/aws-cloudformation/cloudformation-guard/issues/93 ?

**Comment by 0xjjoyy** [comment]
Is junitxml output so we can integrate with CodeBuild reports also on the roadmap?

**Comment by dchakrav-github** [comment]
Current set of output-formats supported as a part of PR #156. Supports JSON, YAML and SingleLine formats. Does not have XML or other representation support. Contributions welcome post PR merge :)

**Comment by gandhek** [comment]
We have released a new minor version of Guard 2.0 which supports various output formats that @dchakrav-github mentioned. You can find the newest version at - https://github.com/aws-cloudformation/cloudformation-guard/releases/tag/2.0.3
